test-cases/performance/perf-regression-nosqlbench-latency-mixed-50r-50w.yaml

```diff
@@ -0,0 +1,3 @@
+stress_cmd_m: "nosqlbench cql-tabular2 rampup-cycles=1M main-cycles=50M write_ratio=5 read_ratio=5 threads=150 striderate=10 strides=800 --progress console:120s rf=3 partsize=5000 driver=cql-scylla -v"
+user_prefix: 'perf-latency-nosqlbench-aws-mixed'
```

Comment:
This should be in configuration because it contains the name of configuration.

Reply:
Moved to test yaml